DuckDB.NET.Data/DuckDBCommand.cs

```diff
@@ -21,25 +23,32 @@ public class DuckDbCommand : DbCommand
     public override bool DesignTimeVisible { get; set; }
     public override UpdateRowSource UpdatedRowSource { get; set; }
 
+#pragma warning disable CS8765 // Nullability of type of parameter doesn't match overridden member (possibly because of nullability attributes).
     public override string CommandText { get; set; }
 
     protected override DbConnection DbConnection
     {
+#pragma warning disable CS8603 // Possible null reference return.
         get => connection;
+#pragma warning restore CS8603 // Possible null reference return.
         set => connection = (DuckDBConnection)value;
     }
+#pragma warning restore CS8765 // Nullability of type of parameter doesn't match overridden member (possibly because of nullability attributes).
 
-    public DuckDbCommand()
+    public DuckDbCommand() : this(string.Empty, null)
     {
     }
 
-    public DuckDbCommand(string commandText)
+    public DuckDbCommand(string commandText) : this(commandText, null)
```

Comment:
If you are passing `commantText` to another constructor, assignment can be removed here.

Reply:
Should be `string? commandText`